frontend/src/lib/components/LemonButton/index.tsx

```diff
@@ -21,9 +21,21 @@ export interface LemonButtonProps extends LemonButtonPropsBase {
 
 /** Styled button. */
 function LemonButtonInternal(
-    { children, type = 'default', className, popup, to, ...buttonProps }: LemonButtonProps,
+    { children, type, className, popup, to, ...buttonProps }: LemonButtonProps,
     ref: React.Ref<JSX.IntrinsicElements['button']>
 ): JSX.Element {
+    const parentPopupId = useContext(PopupContext)
+    if (parentPopupId !== 0) {
+        // If this button is inside a popup, we use more menu-like default props
+        if (!('fullWidth' in buttonProps)) {
+            buttonProps.fullWidth = true
+        }
```

Comment:
I agree, I went for this because I'm a bit lazy :D Initially I wanted to just do a `Menu.Item` component that'd put out a `LemonButton`, but that'd be more effort. I will just manually add the needed props for now, and maybe come back to the `Menu` idea at a later time.